README.md

```diff
@@ -50,3 +56,9 @@
 
 ## Advance ML-on-Code awareness
 
+### App team activities
```

Comment:
Apps team or any other team, doesn't have his own objectives, every objective is linked to a OKR from the Company. 

As I explained in the leads meeting `Advance ML-on-Code awareness` is purely a DevRel OKR